services/provider/server/server.go

```diff
@@ -400,6 +411,44 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	// Fetch noobaa remote secret and management address and append to extResources
+	noobaaOperatorSecret := &v1.Secret{}
+	consumerID := strings.TrimPrefix(consumerResource.Name, "storageconsumer-")
+	if consumerID != "" && len(consumerID) == 0 || consumerID == consumerResource.Name {
+		return nil, fmt.Errorf("failed to get consumerID from consumerResource Name%s %v", consumerResource.Name, err)
+	}
+	noobaaOperatorSecretName := consumerID
+	err = s.client.Get(ctx, types.NamespacedName{Name: noobaaOperatorSecretName, Namespace: s.namespace}, noobaaOperatorSecret)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get %s secret. %v", noobaaOperatorSecretName, err)
+	}
```

Comment:
StorageConsumer name is storageConsumer-clusterID (openshift clusterID) of the client cluster.
Do we want to use consumer k8s UID here to fetch or the openshfit clusterID of the client Cluster?

Reply:
Updated to this format
`noobaa-remote-join-secret-{clusterid}`